src/networks/invertible_network_conditional_glow.jl

```diff
@@ -68,12 +70,13 @@ struct NetworkConditionalGlow <: InvertibleNetwork
     K::Int64
     squeezer::Squeezer
     split_scales::Bool
+    summary_net::Union{FluxBlock,InvertibleNetwork, Nothing}
 end
 
 @Flux.functor NetworkConditionalGlow
 
 # Constructor
-function NetworkConditionalGlow(n_in, n_cond, n_hidden, L, K;freeze_conv=false,  split_scales=false, rb_activation::ActivationFunction=ReLUlayer(), k1=3, k2=1, p1=1, p2=0, s1=1, s2=1, ndims=2, squeezer::Squeezer=ShuffleLayer(), activation::ActivationFunction=SigmoidLayer())
+function NetworkConditionalGlow(n_in, n_cond, n_hidden, L, K; freeze_conv=false,  split_scales=false, summary_net = nothing, rb_activation::ActivationFunction=ReLUlayer(), k1=3, k2=1, p1=1, p2=0, s1=1, s2=1, ndims=2, squeezer::Squeezer=ShuffleLayer(), activation::ActivationFunction=SigmoidLayer())
```

Comment:
How is this different than just the composition `sumarry \odot G`  ?

The whole point of Chain/odot is to easily combine/chain networks and have it work well together, I don't particularly like having networks as input of another like that.

Reply:
The main difference is that composition is not defined for conditional networks. 

Practically: 
I don't think it will work without some extra work because only one of the inputs to G goes into summary. 
(x,y) -> G(x, summary(y))

And summary isnt necessarily invertible so the backward needs to be provide the input to the summary for the gradient calculation (C_save here)
```
Zx, Zy, lgdet = G.forward(X, Y)
dx, x, dy = G.backward(Zx, Zx, Zy; C_save = Y)
```

That said, I agree that it isnt pretty and if these things are easy to solve I am all for it.